BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -407,6 +407,22 @@ Rationale: TLS compression has been subject to security attacks, such as the CRI
 
 Implementers should note that compression at higher protocol levels can allow an active attacker to extract cleartext information from the connection. The BREACH attack is one such case. These issues can only be mitigated outside of TLS and are thus outside the scope of this document. See {{Section 2.6 of RFC7457}} for further details.
       
+### Certificate Compression
+
+Certificate chains often take up the majority of the bytes transmitted during
+the handshake.  In order to manage their size, the following methods can be
+employed:
+
+* Limiting the number of names or extensions;
+* Using keys with small public key representations, like ECDSA; or
```

Comment:
Is this is an inclusive or? That is, can an implementation use some combination of these methods (e.g., ECDSA keys + compression)?

Reply:
good point. It's inclusive and I've tried to make it clear in [9ea2d9b](https://github.com/yaronf/I-D/pull/399/commits/9ea2d9b002e36dfdddb696c55bbadb900260b65c) -- which also includes some clean up